images/scripts/rhel.setup

```diff
@@ -208,7 +208,7 @@ fi
 if [ "${IMAGE#rhel-7}" != "$IMAGE" ] || [ "$IMAGE" == "centos-7" ]; then
     COCKPIT_DEPS="$COCKPIT_DEPS kubernetes-client"
 fi
-if [ "$IMAGE" = "rhel-7-7" ]; then
+if [ "$IMAGE" = "rhel-7-7" ] || [ "$IMAGE" = "rhel-7-8" ]; then
```

Comment:
We could just simplify this -- all our RHEL images have libvirt-dbus now, so it could just go into the original `$COCKPIT_DEPS`, and be blacklisted for centos-7 only. But we can do that later, no reason to rebuild the image for that.